python/test/t_LeastSquaresExpansion_std.py

```diff
@@ -0,0 +1,149 @@
+#! /usr/bin/env python
+
+import openturns as ot
+import openturns.experimental as otexp
+import math as m
+
```

Comment:
Please use the ishigami use case class here too.

Reply:
I understand that you will not update the unit test, because you are short of time until the release. This can be done afterwards, because it is mainly a maintenance issue. This is why I suggest to resolve the conversation, so that we can focus on what matters.